seqr/views/utils/terra_api_utils.py

```diff
@@ -103,8 +103,14 @@ def _safe_get_social(user):
     if not google_auth_enabled() or not hasattr(user, 'social_auth'):
         return None
 
-    social = user.social_auth.filter(provider=SOCIAL_AUTH_PROVIDER)
-    return social.first() if social else None
+    social_auth = user.social_auth.filter(provider=SOCIAL_AUTH_PROVIDER)
```

Comment:
you can clean this up by changing this to `social = user.social_auth.filter(provider=SOCIAL_AUTH_PROVIDER).first()` - `first` will return None if nothing matches so the conditional check that was here was never necessary